dbt/adapters/snowflake/connections.py

```diff
@@ -437,6 +438,16 @@ def execute(
             table = dbt.clients.agate_helper.empty_table()
         return response, table
 
+    def add_standard_query(self, sql, auto_begin, bindings, abridge_sql_log):
```

Comment:
We just added this to `dbt-core` for a similar reason:
https://github.com/dbt-labs/dbt-core/blob/24ca76ea58210c3bec367734dfebfc974b58be21/core/dbt/adapters/sql/connections.py#L158

It might be worth updating the definition in `dbt-core` with defaults that match `add_query()` and then referencing that here.

Reply:
I just re-read the ticket. It looks like we want to backport this into `1.4.latest`, `1.3.latest`, and `1.2.latest`. Is that right? If so, we can't use the new method since that is only in `v1.5`. In that case, I'm fine with this.